src/pages/academy/components/video-banner/carousel/_custom-carousel.tsx

```diff
@@ -25,63 +25,96 @@ import { useRecursiveTimeout } from 'components/hooks/use-recursive-timeout'
 import { Header } from 'components/elements'
 import { Flex } from 'components/containers'
 
-export const PrevButton = ({ color, enabled, is_reviews, onClick, style }) => (
+const StyledDiv = styled.div<{ style?: string; onClick?: () => void }>``
+
+export type ButtonsProps = HTMLAttributes<HTMLDivElement> & {
+    color?: string
+    enabled?: boolean
+    is_enabled?: boolean
+    left?: boolean
+    is_reviews?: boolean
+    disabled?: boolean
+    style?: CustomBlogVideoNavStyleType | string
```

Comment:
Certain can be removed since it is already inside the `HTMLButtonElement` properties. But please check my my code recommendation, might not be named that way